frontend/src/components/grantaccess/GrantDelegatedAdmin.vue

```diff
@@ -136,6 +139,12 @@ const confirmSubmit = async () => {
                 errorList.push(forestClientNumber ?? '');
             }
         });
+        if (returnResponse.data.email_sending_status == EmailSendingStatus.SentToEmailServiceFailure) {
+            setNotificationMsg(
```

Comment:
Did we test the situation when email failed to sent, and there is also an error when grant duplication access? I wonder if the error message will override each other...

Reply:
Hm... I think you are right, I didn't corelated the case. For too long, I think I am losing frontend sense even when I saw previous lines "errorList" and was confused. Need to look into it. Thank you so much!!

Reply:
Ticket #1532 is added for dealing with this message overriden issue.